.github/workflows/dispatch-build-bottle.yml

```diff
@@ -79,15 +82,15 @@ jobs:
       GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
       HOMEBREW_GITHUB_API_TOKEN: ${{secrets.GITHUB_TOKEN}}
     steps:
-      - name: ${{github.event.inputs.formula}}
+      - name: ${{inputs.formula}}
         id: print_details
         run: |
-          echo sender=${{github.event.sender.login}}
-          echo formula=${{github.event.inputs.formula}}
-          echo runner=${{github.event.inputs.runner}}
-          echo timeout=${{github.event.inputs.timeout}}
-          echo issue=${{github.event.inputs.issue}}
-          echo upload=${{github.event.inputs.upload}}
+          echo sender="${{github.event.sender.login}}"
+          echo formula="${{inputs.formula}}"
```

Comment:
Maybe single quotes will be better here to avoid shell expansion which we don't need.

Reply:
Yea, I agree, actually. Will change.